circle.yml

```diff
@@ -1,6 +1,6 @@
 machine:
   xcode:
-    version: 8.2
+    version: 8.3
```

Comment:
Cool, that's a good first step 😉 Unfortunately it will only solve it on CircleCI (which is a good start) but not locally.

The `mdimport /Applications/Xcode*` might be worth a try for local builds